rancher2/schema_app.go

```diff
@@ -0,0 +1,66 @@
+package rancher2
+
+import (
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+//Schemas
+
+func appFields() map[string]*schema.Schema {
+	s := map[string]*schema.Schema{
+		"project_id": &schema.Schema{
+			Type:        schema.TypeString,
+			Required:    true,
+			ForceNew:    true,
+			Description: "Project ID to add app",
+		},
+		"name": &schema.Schema{
+			Type:        schema.TypeString,
+			Required:    true,
+			ForceNew:    false,
```

Comment:
`name` argument should be set as `ForceNew: true` due to app name couldn't be updated once is deployed.